internal/web/routes/routes.go

```diff
@@ -0,0 +1,18 @@
+package routes
```

Comment:
For maintainability, consider grouping similar routes together or using a subrouter for GitHub-related actions. This can improve the organization of your code and make it easier to modify or expand in the future.